packages/react-vapor/src/components/tab/TabSelectors.ts

```diff
@@ -9,11 +9,8 @@ const getTabGroup = (state: IReactVaporState, ownProps: ITabPaneOwnProps) => {
     return _.findWhere(state.tabs, {id: ownProps.groupId ?? DEFAULT_GROUP_ID});
 };
 
-const getTabSelected = (tabId: string) =>
-    createSelector(
-        getTabGroup,
-        (tabGroup: ITabGroupState): boolean => !!_.findWhere(tabGroup.tabs, {id: tabId})?.isSelected
-    );
+const getTabSelected = () =>
+    createSelector(getTabGroup, (tabGroup: ITabGroupState) => _.findWhere(tabGroup.tabs, {isSelected: true}) ?? false);
```

Comment:
Since now we return an object instead of previously a boolean, I think it would be best to return `undefined` when no tab is selected. I will be easier to get the typing right
```suggestion
    createSelector(getTabGroup, (tabGroup: ITabGroupState) => _.findWhere(tabGroup.tabs, {isSelected: true}));
```